.github/labeler.yml

```
@@ -22,7 +22,12 @@ code quality:
           - .pre-commit-config.yaml
           - pyproject.toml
 
-controllers:
+controller backend:
+  - changed-files:
+      - any-glob-to-any-file:
+          - src/controllers/**
```

```suggestion
      - any-glob-to-any-file:
          - src/controllers/**.ts
          - src/controllers/common**
          - src/controllers/lodash.mixxx.js
```

Or check if the first character of the file name is upper case

---

You're thinking of res/controllers, not src/controllers. If not every res/controllers file is a mapping, maybe we should make that less specific instead?

---

You're right - I mixed src and res. I looked for files in res/controllers, which are not specific for a particular mapping.